File: go/easy/two-fer.go
// https://exercism.org/tracks/go/exercises/two-fer
package easy

import "fmt"

func ShareWith(name string) string {
	if name == "" {
		return "One for you, one for me."
	}

	return fmt.Sprintf("One for %s, one for me.", name)
}

File: ts/easy/two-fer.ts
//https://exercism.org/tracks/typescript/exercises/two-fer

/**
 * This stub is provided to make it straightforward to get started.
 */

export function twoFer(name?: string): string {
  if (name == "" || name == null) {
    return "One for you, one for me.";
  }

  return `One for ${name}, one for me.`;
}
